### PR TITLE
Fixed issues that give http 500 error in new relic

### PR DESCRIPTION
**Pull request by cableman**
Ticket: https://jira.itkdev.dk/browse/ER-815

It seams that somethings have changed in datawell 5.2 in relations to records that no longer exits which hits eReolen reference lists and some of the checks and changes we have.

It also seams that the core modules throws some error when error reporting is enabled, but I thing that DDB Core should fix these.

### CODE REVIEW

File: sites/all/modules/overdrive/overdrive.module [hunk 1]
```diff
@@ -125,6 +125,7 @@ function overdrive_ting_covers($entities) {
 function overdrive_ding_entity_buttons($type, $entity) {
   $buttons = array();
 
+//  if (preg_match('/^publizon:/', $entity->ding_entity_id);
```
Comment (rimi-itk) [inline sites/all/modules/overdrive/overdrive.module:128]: This looks like a leftover from a debugging session.

File: sites/all/modules/reol_base/reol_base.module [hunk 2]
```diff
@@ -1730,15 +1731,17 @@ function reol_base_opensearch_post_execute($request, $response, $raw_response) {
       switch (get_class($res)) {
         case 'TingClientObject':
           $res->on_quota = TRUE;
-          if (isset($raw_response->searchResponse->result->searchResult[$raw_index]->collection->object[0]->collection->record)) {
+          if (is_array($raw_response->searchResponse->result->searchResult[$raw_index]->collection->object) &&
```
Comment (rimi-itk) [inline sites/all/modules/reol_base/reol_base.module:1734]: Are we absolutely sure that `$raw_response->searchResponse->result->searchResult` is an array here?

Comment (cableman) [reply sites/all/modules/reol_base/reol_base.module:1734]: jeps

Comment (cableman) [reply sites/all/modules/reol_base/reol_base.module:1734]: if it's not they I think the whole parsing in the tinc-client is f'ed

File: sites/all/modules/overdrive/overdrive.module (hunk 1 above)
Comment (cableman) [reply sites/all/modules/overdrive/overdrive.module]: Removed, tried a little different things to track down the root of the issue.

**Pull request merged**